internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -507,11 +507,14 @@ components:
       type: string
       enum:
         - aws
-        - gcp
         - azure
         - edge-commit
         - edge-container
         - edge-installer
+        - gcp
+        - image-installer
+        - qcow2
```

Comment:
Call this `guest-image`?

Reply:
In RHEL it is called `rhel-guest-image` and since we dropped the `rhel-` prefix from images, the suggestion looks good to me.